prep-html-script-tag/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Script Tag</title>
+  </head>
+  <body>
+    <script>
+      console.log('I have added JS to the script tag in the HTML!');
+    </script>
+    <script src="main.js">
+      console.log("I have added a script file via the src attribute!")
```

Comment:
this can be removed, you only need the src attribute pointing to the main.js file